janusgraph-core/src/main/java/org/janusgraph/graphdb/query/vertex/BasicVertexCentricQueryBuilder.java

```diff
@@ -500,10 +517,25 @@ protected BaseVertexCentricQuery constructQuery(RelationCategory returnType) {
         return query;
     }
 
+    private BaseVertexCentricQuery getAllRelationsQuery() {
+        System.out.println("here");
```

Comment:
Unnecessary line

Reply:
it is a crucial line; the whole boost is coming from here!

Reply:
My mistake! Bring this line back :smile: